client/termsetting/handlers/geneVariant.ts

```diff
@@ -97,6 +98,8 @@ export function fillTW(tw: GeneVariantTW, vocabApi: VocabApi, defaultQ: GeneVari
 		throw 'cannot recognize tw.term.kind'
 	}
 
+	if (!Object.hasOwn(tw.q, 'type')) tw.q.type = 'values'
```

Comment:
my typescript plugin complains about `hasOwn`
```
Property 'hasOwn' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2022' or later.
any
```

Reply:
Addressed in new commit